### DISCUSSION TITLE
v5.0.0

### DISCUSSION THREAD

**Discussion by simurai** [primer/github-vscode-theme#217] [Releases]
## 5.0.0

- Add new "GitHub Dark High Contrast" theme [#203](https://github.com/primer/github-vscode-theme/pull/203).
- Update colors from Primer Primitives [#203](https://github.com/primer/github-vscode-theme/pull/203).

![GitHub VS Code theme](https://user-images.githubusercontent.com/378023/132220037-3cd3e777-55a6-445f-9a2e-da6020ebd78d.png)

<hr /><em>This discussion was created from the release <a href='https://github.com/primer/github-vscode-theme/releases/tag/v5.0.0'>v5.0.0</a>.</em>

**Comment by simurai** [comment]
Reported in https://github.com/primer/github-vscode-theme/issues/220 

---

The Dark Default theme has too much contrast and looks horrible from v5.0.0!

**Comment by simurai** [comment]
Reported in https://github.com/primer/github-vscode-theme/discussions/219

---

### v2 colors in the GitHub Dark Default theme look horrible

The colors used earlier were much better,

**Comment by stellirin** [comment]
Light has recently changed some colours and now the terminal has some parts that are too dark:
<img width="739" alt="image" src="https://user-images.githubusercontent.com/1012095/136839601-c9503465-ac6c-4f43-b836-7bb1dd93b88b.png"> 
<img width="739" alt="image" src="https://user-images.githubusercontent.com/1012095/136839687-6c578c37-8b70-4e57-bd25-e50446854dfa.png">

Compared to Dimmed:
<img width="739" alt="image" src="https://user-images.githubusercontent.com/1012095/136839817-8755bdc8-7102-414f-80a7-a91283238c34.png">
<img width="739" alt="image" src="https://user-images.githubusercontent.com/1012095/136839856-190429b5-12ef-4668-842e-d02fcc73769f.png">

**Reply by dominicegginton** [reply]
You should open a new issue for this. 
**TIP**: You can click on the three dots on the upper right of your post and `Reference in a new issue`
